packages/fast-element/src/interfaces.ts

```diff
@@ -1,7 +1,9 @@
-export type Constructable<T = {}> = {
-    new (...args: any[]): T;
-};
-
 export type Callable = typeof Function.prototype.call | { call(): void };
 
 export const emptyArray = Object.freeze([]);
+
+export type CustomElementConstructor = {
+    new (): HTMLElement;
+};
+
+export type Omit<T, K extends keyof any> = Pick<T, Exclude<keyof T, K>>;

```

Comment:
Are you unable to use the `Omit` language feature?

https://www.typescriptlang.org/docs/handbook/utility-types.html#omittk

Reply:
I thought it wasn't available in the version of TypeScript we were using. I'll try removing this and see if it compiles.

Reply:
Seems to build now that I've removed it. Good call!

Reply:
Yea the packages manage their versions independently - this and fast-components target latest I believe